include/boost/variant/recursive_wrapper.hpp

```diff
@@ -125,8 +126,8 @@ recursive_wrapper<T>::recursive_wrapper(const T& operand)
 
 #ifndef BOOST_NO_CXX11_RVALUE_REFERENCES 
 template <typename T>
-recursive_wrapper<T>::recursive_wrapper(recursive_wrapper&& operand)
-    : p_(new T( detail::variant::move(operand.get()) ))
+recursive_wrapper<T>::recursive_wrapper(recursive_wrapper&& operand) BOOST_NOEXCEPT
+    : p_(boost::exchange(operand.p_, nullptr))
```

Comment:
`nullptr` may not be available. Use 0 :(

Reply:
Fixed